cypress/integration/roamingTest.js

```diff
@@ -1,16 +1,16 @@
-import RoumingPage from '../../pages/RoamingPage'
+import RoamingPage from '../../pages/RoamingPage'
 import * as data from '../../test_data/roaming_data'
 
-describe(" Should verify tariffs in 'Rouming' page", ()=>{
+describe("Verification the titles of roaming tariffs", ()=>{
```

Comment:
Not 'Verification', but 'Verify' - the verb.
`Verify the titles of roaming tariffs`

Or `Titles verification for roaming tariffs`